starter_code/elevator.js

```diff
@@ -1,19 +1,86 @@
 class Elevator {
-  constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
-  }
-
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
-  _passengersLeave() { }
-  floorUp() { }
-  floorDown() { }
-  call() { }
-  log() { }
+  constructor() {
```

Comment:
Utiliza el constructor para inicializar los atributos del objecto no lo dejes fijo. 
constructor(param1, param2, param3) {
this.puertasDelAscensor = param1 
Y así con todos así es más flexible. 
}